src/extension/features/accounts/hide-memo-col/README.md

```diff
@@ -0,0 +1,5 @@
+# Hide the Memo Column for Google Chrome
```

Comment:
```suggestion
# Hide the Memo Column
```

I think you meant to delete the "for Google Chrome"

Reply:
🤦🏼‍♂️ yup, sure did.  Thanks for catching that, lol.  I'll take note and adjust this when I dive back into solving the storage issue I'm having.